scripts/helpers/buildArticlePage.js

```diff
@@ -23,7 +23,7 @@ const buildArticleTags = () => {
   tagsWrapper.classList.add("tags-aside");
 
   const heading = document.createElement("h2");
-  heading.classList.add("util-body-s");
+  heading.classList.add("util-title-s");
```

Comment:
What are the main differences between title and body? I do like using non-body styles for headings.